roomacoustics/sti.py

```diff
@@ -0,0 +1,176 @@
+"""
+Returns a np array with the female or male STI (speech transmission index), a 
+single number value on a metric scale between 0 (bad) and 1 (excellent) for 
+quality assessment of speech transmission channels.
+
+The indices are based on the modulation transfer function (MTF) that determinines
+affections of the intensity envelope throughout the transmission. The MTF values
+are assessed from the IR and are further modified based on auditory, ambient 
+noise and masking aspects.
+
+STI considers 7 octaves between 125 Hz and 8 kHz (125 Hz is not considered for
+the female STI) and 14 modulation frequencies between 0.63 Hz and 12 Hz. 
+
+
+============================
+
+Reference
+---------
+..  [1] IEC 60268-16:2011
+    Sound system equipment - Part 16: Objective rating of speech intelligibility 
+    by speech transmission index
+
+"""
+
+import warnings
+from pyfar import Signal  
+import pyfar.dsp.filter as filt
+import numpy as np
+
+def sti(signal, data_type=None, gender='male', level=np.array([np.nan]), 
+            snr=np.array([np.nan])):
```

Comment:
- I would use `level=None` and `snr=None` as default - might be more pythonic
- `level` is named `noise` in the docstring :)
- I would suggest an additional convenience parameter `pad=True`. If pad is true, sti() zero pads the impulse response to 1.6 seconds if it is shorter.